diff -uNr libtorrent-0.13.7/scripts/common.m4 libtorrent-0.13.7.mod/scripts/common.m4
--- libtorrent-0.13.7/scripts/common.m4	2018-06-07 07:24:43.000000000 +0300
+++ libtorrent-0.13.7.mod/scripts/common.m4	2019-07-03 20:56:25.477890400 +0300
@@ -150,21 +150,6 @@
   ])
 ])
 
-AC_DEFUN([TORRENT_CHECK_EXECINFO], [
-  AC_MSG_CHECKING(for execinfo.h)
-
-  AC_RUN_IFELSE([AC_LANG_SOURCE([
-      #include <execinfo.h>
-      int main() { backtrace((void**)0, 0); backtrace_symbols((char**)0, 0); return 0;}
-      ])],
-    [
-      AC_MSG_RESULT(yes)
-      AC_DEFINE(USE_EXECINFO, 1, Use execinfo.h)
-    ], [
-      AC_MSG_RESULT(no)
-  ])
-])
-
 AC_DEFUN([TORRENT_CHECK_ALIGNED], [
   AC_MSG_CHECKING(the byte alignment)
 
